src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-postgres-ddl.sql

```diff
@@ -12,15 +12,15 @@ CREATE TABLE order_line (
   ol_supply_w_id int NOT NULL,
   ol_quantity decimal(2,0) NOT NULL,
   ol_dist_info char(24) NOT NULL,
-  PRIMARY KEY (ol_w_id,ol_d_id,ol_o_id,ol_number)
+  PRIMARY KEY ((ol_w_id,ol_d_id) hash,ol_o_id,ol_number)
 );
 
 DROP TABLE IF EXISTS new_order;
 CREATE TABLE new_order (
   no_w_id int NOT NULL,
   no_d_id int NOT NULL,
   no_o_id int NOT NULL,
-  PRIMARY KEY (no_w_id,no_d_id,no_o_id)
+  PRIMARY KEY ((no_w_id,no_d_id) hash,no_o_id)
 );
 
 DROP TABLE IF EXISTS stock;
```

Comment:
Please also update `stock` table's PK to be (`s_w_id,s_i_id` hash)

Reply:
Done.